src/pkg/scan/dao/scan/model.go

```diff
@@ -43,3 +43,68 @@ func (r *Report) TableUnique() [][]string {
 		{"digest", "registration_uuid", "mime_type"},
 	}
 }
+
+// VulnerabilityRecord of an individual vulnerability. Identifies an  individual vulnerability item in the scan.
+// Since multiple scanners could be registered with the projects, each scanner
+// would have it's own definition for the same CVE ID. Hence a CVE ID is qualified along
+// with the ID of the scanner that owns the CVE record definition.
+// The scanner ID would be the same as the RegistrationUUID field of Report.
+// Identified by the `cve_id` and `registration_uuid`.
+// Relates to the image using the `digest` and to the report using the `report UUID` field
+type VulnerabilityRecord struct {
+	ID               int64   `orm:"pk;auto;column(id)"`
+	CVEID            string  `orm:"column(cve_id)"`
+	RegistrationUUID string  `orm:"column(registration_uuid)"`
+	Digest           string  `orm:"column(digest)"`
+	Report           string  `orm:"column(report_uuid)"`
+	Package          string  `orm:"column(package)"`
+	PackageVersion   string  `orm:"column(package_version)"`
+	PackageType      string  `orm:"column(package_type)"`
+	Severity         string  `orm:"column(severity)"`
+	Fix              string  `orm:"column(fixed_version);null"`
+	URL              string  `orm:"column(urls);null"`
+	CVE3Score        float64 `orm:"column(cve3_score);null"`
```

Comment:
Let's use consistent naming for CVSS info columns:
cvss_score_v3
cvss_score_v2
cvss_vector_v3
cvss_vector_v2

Reply:
Done